aws-connect

```diff
@@ -161,6 +161,17 @@ if [ -z "${tag_value}" ] && [ -z "${instance_id}" ]; then
   exit 1
 fi
 
+# If no region parameter passed then try to find one otherwise we default
+if [ -z "${aws_region:+x}" ]; then
+  if [ -n "${aws_profile}" ]; then
+    aws_region=$(aws --profile "${aws_profile}" configure get region)
```

Comment:
Hmm.  This doesn't return the region for me (using aws cli v1 and with a region configured for my profile in ~/.aws/config. 

```
[default]
region=us-east-1

[staging]
region=us-west-2

[production]
region=us-east-1
```
![image](https://user-images.githubusercontent.com/6616703/96624245-7b8de780-12da-11eb-9e37-6c2b5a07ac0a.png)

Maybe check if aws_region is set after this and if not, default it to `default_aws_region`?

Reply:
When creating named profiles in the `.aws/config` file you need to prefix them with `profile`. This differs from the `.aws/credentials` file.

```
[default]
region=us-east-1

[profile staging]
region=us-west-2

[profile production]
region=us-east-1
```

Reply:
I've updated it so it properly cascades. It makes sense since a profile could exist but no region actual set for it.

Reply:
Ahhhh yes, I remember this now.  #$@#$ aws config files!  But yeah, good to fix this as well in case there is no region set.